feder/assets/scss/style.scss

```diff
@@ -77,6 +77,43 @@ $gray: #F0F0F0;
   .page-title {
     color: $red;
   }
+  .email_metadata {
+
+  }
+
+  .metadata-button-container {
+     text-align: right;
+    >.navbar-toggle {
+      padding-top: 2px;
+      padding-bottom: 2px;
+      color: $btn-primary-color;
+      background-color: $btn-primary-bg;
+      border-color: $btn-primary-border;
+      float: none;
+      display: inline-block; // e.g.  display regardless of screen size
```

Comment:
@mik-laj , czy ten blok kodu to nie duplikacja bootstrapa? W jakiej części?

Reply:
usunąłem i użyłem btn-primary zamiast tego